lib/model.js

```diff
@@ -314,6 +314,25 @@ Model._getAccessTypeForMethod = function(method) {
 
   var ACL = Model._ACL();
 
+  // Check the explicit setting of accessType
+  if (method.accessType) {
+    assert(method.accessType === ACL.READ ||
```

Comment:
Can we avoid hard coding these? I know we have the enum... but that is already a bit static. Custom access types might be nice in the future.